includes/class-hcard-user.php

```diff
@@ -204,16 +206,16 @@ public static function save_profile( $user_id ) {
 			return false;
 		}
 		$fields = array_merge( self::extra_fields(), self::address_fields() );
-		$p      = array_filter( $_POST );
+		$p      = array_filter( $_POST ); // phpcs:ignore
 		foreach ( $fields as $key => $value ) {
 			if ( isset( $p[ $key ] ) ) {
 				update_user_meta( $user_id, $key, sanitize_text_field( $p[ $key ] ) );
 			} else {
 				delete_user_meta( $user_id, $key );
 			}
 		}
-		if ( isset( $_POST['relme'] ) ) {
-			$relme = explode( "\n", $_POST['relme'] );
+		if ( isset( $_POST['relme'] ) ) { // phpcs:ignore
```

Comment:
Isn't there a requst/query object that we could use instead?

Reply:
There is $_REQUEST, I suppose, but no function.

Reply:
no WordPress "native" solution?

Reply:
There probably should be one that doesn't have people touching the $_POST variable, but no. I should suggest that.